src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +70,81 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+}
+
+/**
+ * Represents the Block component of a Person's address in the address book.
+ */
+class Block{
+	private String block;
+	
+	public Block(String block){
+		this.block = block;
+	}
+	
+	public String getBlock() {
+		return block;
+	}
+
+	public void setBlock(String block) {
+		this.block = block;
+	}
+}
+
+/**
+ * Represents the Street component of a Person's address in the address book.
+ */
+class Street{
+	private String street;
+	
+	public Street(String street){
+		this.street = street;
+	}
+	
+	public String getStreet() {
+		return street;
+	}
+
+	public void setStreet(String street) {
+		this.street = street;
+	}
+}
+
+/**
+ * Represents the Unit component of a Person's address in the address book.
+ */
+class Unit{
+	private String unit;
+	
+	public Unit(String unit){
+		this.unit = unit;
+	}
+	
+	public String getUnit() {
+		return unit;
+	}
+
+	public void setUnit(String unit) {
+		this.unit = unit;
+	}
+}
+
+/**
+ * Represents the Postal Code component of a Person's address in the address book.
+ */
+class PostalCode{
+	private String postalCode;
+	
+	public PostalCode(String postalCode){
+		this.postalCode = postalCode;
+	}
+	
+	public String getPostalCode() {
+		return postalCode;
+	}
+
+	public void setPostalCode(String postalCode) {
+		this.postalCode = postalCode;
+	}
```

Comment:
You have not updated the tests or the documents to reflect the changes. These needs to be updated as well.